### DISCUSSION TITLE
Error in nextra when doing latex

### DISCUSSION THREAD

**Discussion by mwala-joel** [shuding/nextra#2907] [General]
I am having this error [Could not parse expression with acorn](https://github.com/micromark/micromark-extension-mdx-expression/tree/main/packages/micromark-extension-mdx-expression#could-not-parse-expression-with-acorn) and i dont know how to solve it. I have a lot of math equations that involve `ax` somehow so those are getting highlighted and the whole page doesnt compile. How can i deal with it. Or how to make the compiler to leave the math stuff to katex only 
<img width="529" alt="Screenshot 2024-05-28 162542" src="https://github.com/shuding/nextra/assets/107686531/ec5a4b3a-63dd-48fc-bcae-4d0940cd81c8">
<img width="512" alt="Screenshot 2024-05-28 162721" src="https://github.com/shuding/nextra/assets/107686531/05ce8974-294b-4f69-82ad-1a65fba6cfff">
<img width="739" alt="Screenshot 2024-05-28 162830" src="https://github.com/shuding/nextra/assets/107686531/12508b86-a5f3-4a19-8745-b1d699eb8cbc">

**Comment by dimaMachina** [comment]
https://the-guild.dev/blog/nextra-3#-syntax-no-longer-works-for-latex
